src/main/cli/cli.c

```diff
@@ -5250,17 +5251,17 @@ static void cliResource(char *cmdline)
 #ifdef USE_TIMER_MGMT
 static void printTimerDetails(const ioTag_t ioTag, const unsigned timerIndex, const bool equalsDefault, const dumpFlags_t dumpMask, printFn *printValue)
 {
-    const char *format = "timer %c%02d %d";
+    const char *format = "timer %c%02d af%d";
```

Comment:
Looks a bit strange to see `af` in lower case. Did we have any rules/standards for capitalization?

Reply:
In a way I'd prefer to have all of the `diff` output be lower case, to not create the impression that CLI input is case sensitive. But I suspect this train has left the station a long time ago, so I'll change.